src/Microsoft.Extensions.ProjectModel.Sources/IProjectContext.cs

```diff
@@ -24,5 +25,7 @@ public interface IProjectContext
         IEnumerable<string> CompilationItems { get; }
         IEnumerable<string> EmbededItems { get; }
         string FindProperty(string propertyName);
+        IEnumerable<DependencyDescription> Dependencies { get;}
+        IEnumerable<ResolvedReference> ResolvedReferences { get; }
```

Comment:
Yes, it is sum of all. But it points to the assemblies. Not to the packages/ projects.